### PR TITLE
[graf3d] Port to Win64 (replace Long_t by Longptr_t + pointer formatt…

### PR DESCRIPTION
**Pull request by bellenot**
…ing)

### CODE REVIEW

File: graf3d/eve/src/TEveDigitSet.cxx [hunk 1]
```diff
@@ -410,9 +410,9 @@ void TEveDigitSet::DigitSelected(Int_t idx)
 
 void TEveDigitSet::SecSelected(TEveDigitSet* qs, Int_t idx)
 {
-   Long_t args[2];
-   args[0] = (Long_t) qs;
-   args[1] = (Long_t) idx;
+   Longptr_t args[2];
+   args[0] = (Longptr_t) qs;
+   args[1] = (Longptr_t) idx;
```
Comment (linev) [inline graf3d/eve/src/TEveDigitSet.cxx:416]: Does `TQObject::Emit()` method was adjusted?
According to docu https://root.cern/doc/master/classTQObject.html#a61ac18c7a8216bf941add95d7726268a multiple arguments should be converted to `Long_t`. Did you check this?

Comment (bellenot) [reply graf3d/eve/src/TEveDigitSet.cxx:416]: The `TQxxx` classes were changed a while ago, for example in [this merged PR](https://github.com/root-project/root/pull/7778/files). But you're right, I'll change the documentation of `TQObject`. Thanks @linev !

Comment (bellenot) [reply graf3d/eve/src/TEveDigitSet.cxx:416]: @linev https://github.com/root-project/root/pull/8411

**Review by linev — APPROVED** [review]
You can ignore webgui_ping failure, I have to investigate it on the machine

**Review by couet — APPROVED** [review]

**Pull request merged**